edge/pkg/edgestream/tunnel.go

```diff
@@ -165,6 +165,10 @@ func (s *TunnelSession) Serve() error {
 			return err
 		}
 
+		if mess.MessageType == stream.MessageTypeCloseConnect {
+			return fmt.Errorf("update NodeKubeletEndpoint failed, error: %s", string(mess.Data))
```

Comment:
For the edge, it can only find the problem of closing the connection from the returned information, and cannot clearly indicate that this error is caused by `update NodeKubeletEndpoint`